docs(Azure redis cluster) (#1182)
kyma-project/cloud-manager: docs/user/00-50-redis-cluster.md

> # Redis

Reviewer: ```suggestion
# Redis Cluster
```

> Use the Cloud Manager module to provision Redis cluster.

Reviewer: ```suggestion
Use the Cloud Manager module to provision a Redis cluster.
```

The Cloud Manager module allows you to provision a cloud provider-managed Redis cluster in cluster mode within your cluster network.

> [!NOTE]
> Using the Cloud Manager module and enabling Redis, introduces additional costs. For more information, see [Calculation with the Cloud Manager Module](https://help.sap.com/docs/btp/sap-business-technology-platform-internal/commercial-information-sap-btp-kyma-runtime?state=DRAFT&version=Internal#loioc33bb114a86e474a95db29cfd53f15e6__section_cloud_manager).

## Cloud Providers

When you create a Redis cluster in SAP BTP, Kyma runtime, you depend on the cloud provider of your Kyma cluster. The cloud provider in use determines the exact implementation.

The Cloud Manager module supports the Redis cluster feature of the following cloud providers:

> * Microsoft Azure [Azure Cache for Redis](https://azure.microsoft.com/en-us/products/cache)

Reviewer: I guess AWS and GCP are also supported. I can update it later though -> https://github.com/kyma-project/cloud-manager/issues/1296.

You can configure Cloud Manager's Redis clusters using a dedicated Redis cluster custom resource (CR) corresponding with the cloud provider for your Kyma cluster, namely AzureRedisCluster CR. For more information, see [Redis Resources](./resources/README.md#redis-resources).

### Tiers

> When you provision a Redis cluster, tier is Premium.

Reviewer: ```suggestion
When you provision a Redis cluster, you must use the Premium tier. 
```
